great_expectations/data_context/data_context/serializable_data_context.py

```diff
@@ -87,6 +87,16 @@ def _save_project_config(self) -> None:
         try:
             with open(config_filepath, "w") as outfile:
                 self.config.to_yaml(outfile)
+
+            zep_datasources = self._synchronize_zep_datasources()
+            if zep_datasources:
+                logger.info(
+                    f"Saving {len(self.zep_config.datasources)} ZEP Datasources to {config_filepath}"
+                )
+                with open(config_filepath, "r+") as io_file:
+                    yaml_dict = yaml.load(io_file)
+                    yaml.dump({**yaml_dict, **self.zep_config._json_dict()}, io_file)
+
```

Comment:
so the parent method just calls `self.variables.save_config()` - could we leverage that in any way?

Reply:
Nevermind read the rest of the PR 😬

Reply:
I don't follow.